lib/preprocessor.py

```diff
@@ -361,4 +360,7 @@ def cleanup_nondefault(self):
             v.cleanup()
 
     def kill_current_test(self):
-        self.servers['default'].kill_current_test()
+        for server in self.servers:
+            if server == 'default':
+                continue
+            self.servers[server].kill_current_test()
```

Comment:
The idea was we have the one common 'current test' per cluster. Don’t get this change.